### ISSUE TITLE
[BUG] Select type form items don't validate whether there are values

### ISSUE THREAD

**Issue by andrew-m-j** [danielo515/obsidian-modal-form#188]
**Describe the bug**
When making a form with a select type input and static options, there isn't a check to ensure that values are entered for each choice. If these are omitted, the form submission returns an empty result.

**Expected behavior**

When a select input is used, the form validation should check whether there are values entered for each choice and report an error similar to what happens if a form name is missing.

**Comment by danielo515** [comment]
Yes, this is something I have been postponing. I agree it will be a nice Q0L improvement
